tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_preprocessor.cpp

```diff
@@ -65,37 +66,41 @@ void RejitPreprocessor<RejitRequestDefinition>::ProcessTypeDefForRejit(const Rej
             continue;
         }
 
-        // Compare if the current mdMethodDef contains the same number of arguments as the
-        // instrumentation target
         const auto numOfArgs = functionInfo.method_signature.NumberOfArguments();
-        if (numOfArgs != target_method.signature_types.size() - 1)
+        auto is_exact_signature_match = GetIsExactSignatureMatch(definition);
```

Comment:
@GreenMatan @dudikeleti to implement this feature, I need to add a flag to bypass signature verification for this new integration. What do you think of this addition to the `RejitPreprocessor` API? Are there any changes you'd propose so it can work well with the Debugger?

Reply:
To be honest, it's exactly what I planned to add.
In the Debugger we also need, in specific cases, to have that ability of bypassing signature matching to instrument all the overloads.
I'm in favor of the `GetIsExactSignatureMatch` addition :+1:.
The only thing is to perhaps return `true` for now in the Debugger (instead of `false`) to keep the behavior of the Debugger intact?

Reply:
Sounds good, thanks for the feedback. I changed the return value to `true` in b7ab9c3cb02addc8be2c70db7bcd55893ef73464